doc/source/dev/roadmap.rst

```diff
@@ -50,13 +50,14 @@ requested in this respect.
 Statistics enhancements
 -----------------------
 
-The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
-particularly high importance to the project.
-
-- Improve the options for fitting a probability distribution to data.
-- Expand the set of hypothesis tests.  In particular, include all the basic
-  variations of analysis of variance.
-- Add confidence intervals for all statistical tests.
+The following `scipy.stats` enhancements and those listed in the
+:ref:`scipy-roadmap-detailed` are of particularly high importance to the
+project.
+
+- Overhaul the univariate distribution infrastructure to address longstanding
+  issues (e.g. see `gh-15928 <https://github.com/scipy/scipy/issues/15928>`_.)
```

Comment:
My initial response was similar to @josef-pkt's on that issue: this will be quite challenging to design, and it's not clear before having a prototype that the benefits are enough to justify maintaining two sets of APIs in parallel. That's not actually what the text says here though, and there's certainly significant scope for improvement, so I think the current phrasing here is fine. That said, I thought it was important to point out that committing to a duplicate set of APIs is not a thing that this roadmap addition commits us to.